spec/controllers/wishlists_controller_spec.rb

```diff
@@ -100,15 +101,84 @@
         it "redirects to the wishlists index page" do
           is_expected.to redirect_to(wishlists_path)
         end
+      end
+    end
+
+    describe "GET #edit" do
+      context "when wishlist exists" do
+        subject! { get_edit(id: wishlist.id) }
+
+        it "returns 200 OK" do
+          expect(response).to have_http_status(:ok)
+        end
+
+        it "renders the 'edit' template" do
+          expect(response).to render_template(:edit)
```

Comment:
The same question for You (post, delete). :) Maybe the sameness is a good option but I think that it's not too important and good to know that there are many instructions of the same purpose.